terra-batch-libchart/values.yaml

```diff
@@ -4,7 +4,7 @@ exports:
       name: coa-cromwell-svc
       image: broadinstitute/cromwell:85-46a7918
       port: 8000
-      coaEnabled: true
+      enabled: true
```

Comment:
Changed `coaEnabled` to just `enabled` in the lib chart, and make it control both sub-charts. Default all to `true` -- Leo will enable/disable a different set of services depending on environment/cloud.